keylime/shared_data.py

```
@@ -137,8 +156,20 @@ def __init__(self) -> None:
-        # The Manager must be started BEFORE any fork() calls
+        # Ensure /var/run/keylime exists with correct permissions
+        # before forking the Manager server process.
+        _ensure_runtime_dir()
+        self._socket_path = os.path.join(_RUNTIME_DIR, f"shared_data.{os.getpid()}.sock")
+
+        # Remove stale socket from a previous run (e.g. after a crash).
+        # CPython's SocketListener does not pre-unlink before bind().
+        try:
+            os.unlink(self._socket_path)
+        except (FileNotFoundError, PermissionError):
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(FileNotFoundError, PermissionError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(FileNotFoundError, PermissionError): ...`

(SIM105)